source/core/parameters_lagrangian.cc

```diff
@@ -863,6 +883,8 @@ namespace Parameters
               std::runtime_error("Invalid particle-wall contact force model "));
           }
 
+        dmt_cut_off_threshold = prm.get_double("DMT cut-off threshold");
```

Comment:
I might have an idea, but it's probably overkilled.
Since you have `force_calculation_threshold_distance` straight at the initialization, you can git it with a getter and check that in the `input_parameter_inspector`. But I think the threshold will always be greater no? Like the cut-off will never be more than dp?